extensions/PhabBugz/lib/Util.pm

```diff
@@ -382,4 +387,46 @@ sub request {
     return $result;
 }
 
+sub get_security_sync_groups {
+    my $bug = shift;
+
+    my $phab_sync_groups = Bugzilla->params->{phabricator_sync_groups}
+        || ThrowUserError('invalid_phabricator_sync_groups');
+    my $sync_group_names = [ split('[,\s]+', $phab_sync_groups) ];
+
+    my $bug_groups = $bug->groups_in;
+    my $bug_group_names = [ map { $_->name } @$bug_groups ];
+
+    my @set_groups = intersect($bug_group_names, $sync_group_names);
+
+    return @set_groups;
+}
+
+sub add_security_sync_comments {
+    my ($revisions, $bug) = @_;
+
+    my $phab_error_message = 'Revision is being made private due to unknown Bugzilla groups.';
+
+    foreach my $revision (@$revisions) {
+        add_comment_to_revision( $revision->{phid}, $phab_error_message );
+    }
+
+    my $num_revisions = 0 + @$revisions;
```

Comment:
Use:

my $num_revisions = scalar @$revisions;

Reply:
You can just use bare `@num_revisions` below, since all uses are scalar context, and drop 414 `my $num_revisions` entirely.

`@num_revisions > 1`
`? @num_revisions.' revisions were'`

are both valid.